webrtc.html

```diff
@@ -6611,24 +6625,25 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           </dl>
         </section>
       </div>
-      <p>The <dfn>RTCRtpContributingSource</dfn> objects contain information
-      about a given contributing source, including the most recent time a
+      <p>The <dfn>RTCRtpContributingSource</dfn> and
+      <dfn>RTCRtpSynchronizationSource</dfn> objects contain information about
+      a given contributing source (CSRC) or synchronization source (SSRC),
+      including the most recent time a
       packet that the source contributed to was played out. The browser MUST
       keep information from RTP packets received in the previous 10 seconds.
       When the first audio frame contained in an RTP packet is delivered to the
       <code><a>RTCRtpReceiver</a></code>'s <code><a>MediaStreamTrack</a></code>
       for playout, the relevant <code><a>RTCRtpContributingSource</a></code>
-      objects are updated. The
-      <code><a>RTCRtpContributingSource</a></code> object corresponding to the
-      SSRC is always updated, and if the RTP packet contains CSRCs, then the
-      <code><a>RTCRtpContributingSource</a></code> objects corresponding to
-      those CSRCs are also updated.</p>
+      and <code><a>RTCRtpSynchronizationSource</a></code> objects are updated. The
+      <code><a>RTCRtpSynchronizationSource</a></code> object corresponding to the
+      SSRC identifier is always updated, and if the RTP packet contains CSRC
+      identifiers, then the <code><a>RTCRtpContributingSource</a></code>
+      objects corresponding to those CSRC identifiers are also updated.</p>
```

Comment:
SSRC and CSRC numbers are independent of each other? You could have one RTP packet (ssrc:1, csrc:[1,...]) and then another one (ssrc:2, csrc:[1,...])? Does it make sense to have a reference to the most recent RTCRtpSynchronizationSource  from an RTCRtpContributingSource?

Hmm actually that could be misleading because the SSRC could be updated and so a CSRC's SSRC could represent a different state and time than the CSRC. But what about a source id? Or is this not useful?

It would seem the SSRC's audioLevel would be enough (CSRC's audioLevel being a copy) if it wasn't for the reference representing a different state and time. Copy needed to have audioLevel for that state/time.